packages/core/src/components/essentials/title-meter.ts

```diff
@@ -108,7 +252,7 @@ export class MeterTitle extends Title {
 	 * Appends the associated percentage to the end of the title
 	 */
 	appendPercentage() {
-		const dataValue = this.model.getDisplayData().value;
+		const dataValue = this.model.getDisplayData()[0].value;
```

Comment:
again we need to use `Tools.getProperty()` to avoid errors when `this.model.getDisplayData()` is empty or `this.model.getDisplayData()[0]` doesn't have a `value` key